Makefile

```diff
@@ -2,24 +2,46 @@
 DOCDIR = /usr/share/doc/packages
 VERSION ?= $(shell (git describe 2>/dev/null || echo '0.0.0') | sed -e 's/^v//' -e 's/-/+/' -e 's/-/./')
 
-OS=$(shell source /etc/os-release ; echo $$ID)
+PYTHON_DEPS=python3-setuptools python3-click python3-tox
+PYTHON=python3
+PIPCMD=""
+
+OS=$(shell source /etc/os-release 2>/dev/null ; echo $$ID)
 ifeq ($(OS), opensuse)
 USER=salt
 GROUP=salt
 PKG_INSTALL=zypper -n install
+PYTHON_DEPS=python-pip python-setuptools python-click python-tox
+PYTHON=python
+PIPCMD=pip install click tox
 else
 ifeq ($(OS), sles)
 USER=salt
 GROUP=salt
 PKG_INSTALL=zypper -n install
+PYTHON_DEPS=python-pip python-setuptools
+# sles12.3 in GCP cannot find python-click/tox packages (pip is alternative).
+PYTHON=python
+PIPCMD=pip install click tox
 else
 USER=root
 GROUP=root
 ifeq ($(OS), centos)
 PKG_INSTALL=yum install -y
+PYTHON_DEPS=python-setuptools python-click python-tox
+PYTHON=python
```

Comment:
Here you hardcoded python to be the main version. I think in your case it will default to python3.

However, this statement does not hold on newer SLE distrubutions where python disappeared and you have to explicitly call python3

Reply:
Hi @jschmid1  
CentOS is correct.  But yes,  python3 is valid choice for SuSE and Fedora - updating commit.
- SLES + OPENSUSE => Python3.x.x
- CENTOS7 = python2.7.5
- FEDORA => python 2.7.14, 3.6.2

Reply:
Done!!